### DISCUSSION TITLE
PB bluetooth et appariement robots

### DISCUSSION THREAD

**Discussion by eferigo** [robot-soccer-kit/robot-soccer-kit#52] [Q&A] [answered]
Bonjour,
J'ai toujours des problèmes pour appairer les robots. 
J'utilise un ordi portable sous ubuntu 20.04 et la version du kernel est en 5.13.
Je suis éventuellement disponible cet après-midi vers 13h30 ou 14h pour une assistance en direct.
Merci pour votre aide
Eric FERIGO, lycée Barthou, PAU

**Comment by cgaspard3333** [comment, marked as answer]
Assistance réalisée, problème de congestion de la bande 2.4Ghz très supérieure à la moyenne dans l'établissement (50aine d'appareils Bluetooth détectés)
